sycl/doc/UsersManual.md

```diff
@@ -338,6 +338,11 @@ and not recommended to use in production environment.
 
     NOTE: This flag is currently only supported with the CUDA and HIP targets.
 
+**`-fsycl-implicit-rpath, -fno-sycl-implicit-rpath`**
+
+    Instructs the compiler to add the directory within the clang build or
+    installation containing the SYCL library to the `RPATH` of generated
+    binaries when building in SYCL mode. This flag is enabled by default.
```

Comment:
Which path is going to be used if I have both of them: build and installation directories?

Reply:
It's tied to the clang binary, so it will point to the associated `lib` directory, for example:
```
$ ./build/bin/clang++ -fsycl a.cpp -o a
# RPATH will point to "/path/to/build/lib"
$ ./install/bin/clang++ -fsycl a.cpp -o a
# RPATH will point to "/path/to/install/lib"
```

And when it comes to resolving which libraries to use `RPATH` is looked up before the system libraries, so even if you have DPC++ installed on your system, binaries generated with the `clang++` from your build directory will use the `libsycl.so` from your build directory as well.